### DISCUSSION TITLE
Browser Compatibility?

### DISCUSSION THREAD

**Discussion by evbrew** [ChartsCSS/charts.css#17] [Q&A] [answered]
Are there known browser combability versions with this project?

**Comment by ramiy** [comment, marked as answer]
No issues reported, but you can see the supported browsers list at:

https://github.com/ChartsCSS/charts.css/blob/master/package.json#L60-L66

**Reply by cehwitham** [reply]
Am I right in thinking that means no IE support at all?

**Reply by ramiy** [reply]
You are right, the out-of-the-box version does not support IE. But you can build your own version.

Edit **package.json** and remove `"not ie >= 0",` line.

Then run `npm run build`.

**Reply by evbrew** [reply]
cehwitham - were you able to try this by chance? did it work?
